include/mbedtls/config.h

```diff
@@ -56,6 +56,31 @@
 #define MBEDTLS_HAVE_ASM
 
 /**
+ * \def MBEDTLS_NO_UDBL_DIVISION
+ *
+ * The platform lacks support for double-width integer division (64-bit
+ * division on a 32-bit platform, 128-bit division on a 64-bit platform).
+ *
+ * Used in:
+ *      include/mbedtls/bignum.h
+ *      library/bignum.c
+ *
+ * The bignum code uses double-width division to speed up some operations.
+ * Double-width division is often implemented in software that needs to
+ * be linked with the program. The presence of a double-width integer
+ * type is usually detected automatically through preprocessor macros,
+ * but the automatic detection cannot know whether the code needs to
+ * and can be linked with an implementation of division for that type.
+ * By default division is assumed to be usable if the type is present.
+ * Uncomment this option to prevent the use of double-width division.
+ *
+ * Note that division for the native integer type is always required.
+ * Furthermore, a 64-bit type is always required even on a 32-bit
+ * platform, but it need not support multiplication or division.
+ */
```

Comment:
See “7.18.1.1 Exact-width integer types” in C99 TC2 (n1124) or the corresponding section in other versions (starting in C99).

> The typedef name uint*N*_t designates an unsigned integer type with width *N* .

The standard does not specify that if an *N*-bit type exists then uint*N*_t must exist. @andresag01 What compilers don't do this?

(We only require C89, not C99, and C89 does not define any of the uint*N*_t types. However, we do require a few C99 features, in particular `stdint.h` which is where `uint32_t`, `uint64_t`, and optionally `uint128_t` are defined.)